stdlib/log-api/src/main/ballerina/log/natives.bal

```diff
@@ -18,25 +18,30 @@
 # Logs the specified value at DEBUG level.
 #
 # + msg - The message to be logged
-public extern function printDebug(string msg);
+# + func - The lambda function to be executed, when DEBUG level is enabled
+public extern function printDebug(string msg, (function() returns (string))? func =());
```

Comment:
Slight formatting issue: `func =()` -> `func = ()` 
Needs to be fixed in other places as well.